conftest.py

```diff
@@ -67,6 +67,10 @@ def pytest_ignore_collect(path, config):
             "tests/prophet",
             "tests/pmdarima",
             "tests/diviner",
+            # this test suite has an underscore suffix due to pytorch lightning having an optional
+            # transformers dependency within a utility module that creates an import error within
+            # the python suite `check_mlflow_lazily_imports_ml_packages.py`
+            "tests/transformers_",
```

Comment:
Stack trace (this is an optional import statement within pytorch_lightning for a utility module that fails due to the fact that the script is executing from within a scope namespace where our test module that was named "transformers" resided, which importlib will get a referential match due to the path searching of importlib): 
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/torchmetrics/functional/text/helper_embedding_metric.py", line 26, in <module>
    from transformers import AutoModelForMaskedLM, AutoTokenizer, PreTrainedModel, PreTrainedTokenizerBase
ImportError: cannot import name 'AutoModelForMaskedLM' from 'transformers' (/home/runner/work/mlflow/mlflow/tests/transformers/__init__.py)

Reply:
@BenWilson2 Is this because we have `tests/transformers_/__init__.py`?

Reply:
Renamed `tests/transformers_` to `tests/transformers`, removed `tests/transformers_/__init__.py`, and then ran `python tests/check_mlflow_lazily_imports_ml_packages.py` and it worked.

Reply:
ooof. I didn't even realize that I had added that __init__.. Thanks for verifying.